packages/scripts/src/generate-readme/generator.ts

```diff
@@ -12,6 +12,7 @@ import {
 import { balance } from '@chainlink/ea-factories'
 import { getBalanceTable, inputParamHeaders, paramHeaders } from './tableAssets'
 import { EndpointDetails, EnvVars, IOMap, JsonObject, Package, Schema } from '../shared/docGenTypes'
+import { DEFAULT_WS_API_ENDPOINT } from '../../../sources/tradingeconomics/src/config'
```

Comment:
This isn't needed anymore, right?

Reply:
oh shoot. my IDE loves to autoimport everything :)

Reply:
Lol it happens to me a lot too